share/scan/scan_utils.go

```diff
@@ -56,26 +56,27 @@ var OSPkgFiles utils.Set = utils.NewSet(
 
 var scanErrString = []string{
-	share.ScanErrorCode_ScanErrNone:                "succeeded",
-	share.ScanErrorCode_ScanErrNetwork:             "network error",
-	share.ScanErrorCode_ScanErrNotSupport:          "unsupported OS",
-	share.ScanErrorCode_ScanErrSizeOverLimit:       "file size over limit",
-	share.ScanErrorCode_ScanErrPackage:             "package error",
-	share.ScanErrorCode_ScanErrDatabase:            "database error",
-	share.ScanErrorCode_ScanErrTimeout:             "timeout",
-	share.ScanErrorCode_ScanErrInProgress:          "scan in progress",
-	share.ScanErrorCode_ScanErrRegistryAPI:         "registry API error",
-	share.ScanErrorCode_ScanErrFileSystem:          "access file system error",
-	share.ScanErrorCode_ScanErrContainerAPI:        "container API call error",
-	share.ScanErrorCode_ScanErrXrayAPI:             "Xray API call error",
-	share.ScanErrorCode_ScanErrContainerExit:       "container exit",
-	share.ScanErrorCode_ScanErrAuthentication:      "authentication error",
-	share.ScanErrorCode_ScanErrCertificate:         "certificate error",
-	share.ScanErrorCode_ScanErrCanceled:            "scan canceled",
-	share.ScanErrorCode_ScanErrDriverAPINotSupport: "driver API not supported",
-	share.ScanErrorCode_ScanErrImageNotFound:       "Image not found",
-	share.ScanErrorCode_ScanErrAwsDownloadErr:      "Aws Resource download error",
-	share.ScanErrorCode_ScanErrArgument:            "invalid input arguments",
+	share.ScanErrorCode_ScanErrNone:                  "succeeded",
+	share.ScanErrorCode_ScanErrNetwork:               "network error",
+	share.ScanErrorCode_ScanErrNotSupport:            "unsupported OS",
+	share.ScanErrorCode_ScanErrSizeOverLimit:         "file size over limit",
+	share.ScanErrorCode_ScanErrPackage:               "package error",
+	share.ScanErrorCode_ScanErrDatabase:              "database error",
+	share.ScanErrorCode_ScanErrTimeout:               "timeout",
+	share.ScanErrorCode_ScanErrInProgress:            "scan in progress",
+	share.ScanErrorCode_ScanErrRegistryAPI:           "registry API error",
+	share.ScanErrorCode_ScanErrFileSystem:            "access file system error",
+	share.ScanErrorCode_ScanErrContainerAPI:          "container API call error",
+	share.ScanErrorCode_ScanErrXrayAPI:               "Xray API call error",
+	share.ScanErrorCode_ScanErrContainerExit:         "container exit",
+	share.ScanErrorCode_ScanErrAuthentication:        "authentication error",
+	share.ScanErrorCode_ScanErrCertificate:           "certificate error",
+	share.ScanErrorCode_ScanErrCanceled:              "scan canceled",
+	share.ScanErrorCode_ScanErrDriverAPINotSupport:   "driver API not supported",
+	share.ScanErrorCode_ScanErrImageNotFound:         "Image not found",
+	share.ScanErrorCode_ScanErrAwsDownloadErr:        "Aws Resource download error",
+	share.ScanErrorCode_ScanErrArgument:              "invalid input arguments",
+	share.ScanErrorCode_ScanErrAcquireScannerTimeout: "acquire scanner timeout",
```

Comment:
Is there any background why `ScanErrorCode_ScanErrSignatureScanError` gets skipped?

Reply:
Since it's just one line, I'd recommend adding it, but feel free to skip it if we found it's bigger than that.